api/config/config_test.json

```diff
@@ -69,6 +70,7 @@
     "enableLeoWelder": true,
     "useBillingProjectBuffer": true,
     "unsafeAllowDeleteUser": true,
-    "enableVpcFlowLogs": true
+    "enableVpcFlowLogs": true,
+    "enableVpcServicePerimeter": false
```

Comment:
should we also enable test?

Reply:
Yeah, I think we should go ahead and try it out. Better to find out sooner than later if it breaks something.